<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-1T29DSRM" failures="1" tests="1" name="Default test" time="35.494" errors="0" timestamp="23 פבר׳ 2021 10:51:57 GMT">
  <testcase classname="tests.basic.ShoppingEndToEnd" name="test" time="32.25">
    <failure type="java.lang.RuntimeException" message="java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\Itai\Downloads\IN040972.pdf (The system cannot find the file specified)">
      <![CDATA[java.lang.RuntimeException: java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\Itai\Downloads\IN040972.pdf (The system cannot find the file specified)
at tests.supers.TestBase.onTestFailure(TestBase.java:140)
at tests.basic.ShoppingEndToEnd.test(ShoppingEndToEnd.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\Itai\Downloads\IN040972.pdf (The system cannot find the file specified)
at tests.supers.TestBase.onTestFailure(TestBase.java:140)
at tests.basic.ShoppingEndToEnd.verifyPDFContainsText(ShoppingEndToEnd.java:107)
at tests.basic.ShoppingEndToEnd.startShoppingProcess(ShoppingEndToEnd.java:71)
at tests.basic.ShoppingEndToEnd.test(ShoppingEndToEnd.java:32)
... 25 more
Caused by: java.io.FileNotFoundException: C:\Users\Itai\Downloads\IN040972.pdf (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:212)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:154)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:109)
at util.FileUtilis.isTextInPDFFile(FileUtilis.java:35)
at tests.basic.ShoppingEndToEnd.verifyPDFContainsText(ShoppingEndToEnd.java:101)
... 27 more
]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Default test -->
